src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -0,0 +1,1244 @@
+
+    @HiddenOption
+    @Argument(fullName="justRead",
+                    doc="If true, we won't actually write the output file.  For efficiency testing only", optional=true)
+    public boolean justRead = false;
+
+    /**
+     * If this argument is provided, indels that are larger than the specified size will be excluded.
+     */
+    @Argument(fullName="maxIndelSize", optional=true, doc="Maximum size of indels to include")
+    public int maxIndelSize = Integer.MAX_VALUE;
+
+    /**
+     * If this argument is provided, indels that are smaller than the specified size will be excluded.
+     */
+    @Argument(fullName="minIndelSize", optional=true, doc="Minimum size of indels to include")
+    public int minIndelSize = 0;
+
+    /**
+     * If this argument is provided, select sites where at most a maximum number of samples are filtered at the genotype level.
+     */
+    @Argument(fullName="maxFilteredGenotypes", optional=true, doc="Maximum number of samples filtered at the genotype level")
+    public int maxFilteredGenotypes = MAX_FILTERED_GENOTYPES_DEFAULT_VALUE;
+
+    /**
+     * If this argument is provided, select sites where at least a minimum number of samples are filtered at
+     * the genotype level.
+     */
+    @Argument(fullName="minFilteredGenotypes", optional=true,
+                    doc="Minimum number of samples filtered at the genotype level")
+    public int minFilteredGenotypes = MIN_FILTERED_GENOTYPES_DEFAULT_VALUE;
+
+    /**
+     * If this argument is provided, select sites where a fraction or less of the samples are filtered at
+     * the genotype level.
+     */
+    @Argument(fullName="maxFractionFilteredGenotypes",
+                    optional=true, doc="Maximum fraction of samples filtered at the genotype level")
+    public double maxFractionFilteredGenotypes = MAX_FRACTION_FILTERED_GENOTYPES_DEFAULT_VALUE;
+
+    /**
+     * If this argument is provided, select sites where a fraction or more of the samples are filtered at
+     * the genotype level.
+     */
+    @Argument(fullName="minFractionFilteredGenotypes", optional=true,
+                    doc="Maximum fraction of samples filtered at the genotype level")
+    public double minFractionFilteredGenotypes = MIN_FRACTION_FILTERED_GENOTYPES_DEFAULT_VALUE;
+
+    /**
+     * If this argument is provided, set filtered genotypes to no-call (./.).
+     */
+    @Argument(fullName="setFilteredGtToNocall", optional=true, doc="Set filtered genotypes to no-call")
+    public boolean setFilteredGenotypesToNocall = false;
+
+    @Argument(fullName=StandardArgumentDefinitions.LENIENT_LONG_NAME,
+                            shortName = StandardArgumentDefinitions.LENIENT_SHORT_NAME,
+                            doc = "Lenient processing of VCF files", common = true, optional = true)
+    public boolean lenientVCFProcesssing;
+
+    //@ArgumentCollection
+    //private IntervalArgumentCollection intervalList = new OptionalIntervalArgumentCollection();
```

Comment:
Delete this commented-out argument collection (intervals come from the superclass)